### PR TITLE
Compile aktualizr with asn1c help from yocto

### PR DESCRIPTION
**Pull request by lbonn**
As you'll see, this required a little hack.

Required for https://github.com/advancedtelematic/aktualizr/pull/509 (which is not ready yet)

### CODE REVIEW

File: recipes-sota/asn1c/asn1c.bb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+SUMMARY = "ASN.1 to C compiler"
+DESCRIPTION = "Generates serialization routines from ASN.1 schemas"
+HOMEPAGE = "http://lionet.info/asn1c"
+SECTION = "base"
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=ee8bfaaa7d71cf3edb079475e6716d4b"
+DEPENDS = "asn1c-native"
```
Comment (patrickvacek) [inline recipes-sota/asn1c/asn1c.bb]: This seems like a recursive dependency.

File: recipes-sota/asn1c/files/skeletons_dir_fix.patch [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
++        if(stat(skeletons_dir, &sb)) {
+             /*
+              * The default skeletons directory does not exist,
+              * compute it from my file name:
+-             * ./asn1c/asn1c -> ./skeletons
++             * ./asn1c/asn1c -> ./share/asn1c
+              */
+             char *p;
+             size_t len;
++            printf("HI\n");
```
Comment (patrickvacek) [inline recipes-sota/asn1c/files/skeletons_dir_fix.patch]: Is that printf intentional?

Comment (lbonn) [reply recipes-sota/asn1c/files/skeletons_dir_fix.patch]: obviously not, thanks!

**Comment by cajun-rat** [comment]
I know the asn1c question is still being investigated, but I think we should merge this, and if we end up using protobufs or something else then we can rip it out again.

**Pull request merged**